src/makeCalculator.js

```diff
@@ -4,7 +4,30 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => (calculator.result += num),
+    subtract: (num) => (calculator.result -= num),
+    multiply: (num) => (calculator.result *= num),
+    divide: (num) => {
+      if (num === 0) {
+        return;
+      }
+      calculator.result /= num;
+    },
```

Comment:
It would be a good practice to use `this `instead of object's name

Reply:
> It would be a good practice to use `this `instead of object's name

Good point